test/terra/states/test_aer_state.py

```diff
@@ -161,7 +306,7 @@ def test_appply_unitary(self):
         result = aer_simulator.run(circuit).result()
         expected = result.get_statevector(0)
 
-        state = AerState()
+        state = AerState(method='statevector')
```

Comment:
What happens if we don't have the `method` set now?  I'm concerned that there might be a breaking API change, though I don't remember what we said about whether we consider `AerState` to be part of the public interface or not.

Reply:
`AerState` is not a public interface. Just `AerStatevector` uses and `AerDensitymatrix` will use it.